lib/modules/platform/pr-body.spec.ts

```diff
@@ -12,6 +12,17 @@ describe('modules/platform/pr-body', () => {
       expect(getPrBodyStruct('')).toEqual({
         hash: 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855',
       });
+      expect(
+        getPrBodyStruct(
+          'something \n<!--renovate-debug:eyJjcmVhdGVkQnlSZW5vdmF0ZVZlcnNpb24iOiAiMS4yLjEiLCJ1cGRhdGVkQnlSZW5vdmF0ZVZlcnNpb24iOiAiMS4yLjMifQ==-->'
+        )
+      ).toEqual({
+        hash: '82bc749b41acc2a3a3cc398fbfb0717fd05db152b22e48ac7d27f03a06a5dc3a',
+        debugData: {
+          createdByRenovateVersion: '1.2.1',
+          updatedByRenovateVersion: '1.2.3',
+        },
+      });
     });
```

Comment:
Please, replace last test in the file with these two:file:

```ts
    it('hashes ignoring reviewable section', () => {
      expect(hashBody('foo<!-- Reviewable:start -->bar')).toEqual(
        hashBody('foo')
      );
    });

    it('hashes ignoring debug info', () => {
      expect(hashBody('foo\n<!--renovate-debug:123-->\n')).toEqual(
        hashBody('foo')
      );
    });
```

Reply:
a test like one of those already exists:
```
    it('strips reviewable section', () => {
      expect(getPrBodyStruct('foo<!-- Reviewable:start -->bar')).toEqual({
        hash: hashBody('foo'),
      });
      ```

Reply:
i added the other one, and removed mine.

Reply:
Yes, but I don't like it 🙂

Reply:
Never mind, let's just test our debug info hashing

Reply:
i added your tests , sounds like the best thing to do xD

Reply:
> Never mind, let's just test our debug info hashing

i dont understand what you mean exactly, do u mean we have to change the tests again?

Reply:
i removed my test and now i dont have coverage

lib/modules/platform/pr-body.ts#L53
Added line #L53 was not covered by tests